src/common/geometry.js

```diff
@@ -126,3 +126,15 @@ export const getSliderBounds = (vertices, sliderValue) => {
 
   return { start: start, end: end }
 }
+
+// returns an array of points drawing a circle of a given radius
+export const circle = (radius=1, start=0) => {
+  let points = []
+
+  for (let i=start; i<=128+start; i++) {
+    let angle = Math.PI * 2.0 / 128.0 * i
```

Comment:
You can switch to counterclockwise by multiplying i by -1. I don't think this matters in the mask, but...